DigitalLearningSolutions.Web/Scripts/learningMenu/fullscreen.ts

```diff
@@ -10,7 +10,8 @@ export function setupFullscreen(): void {
 }
 
 export function exitFullscreen(): void {
-  getIFrame()?.classList.remove('fullscreen');
+  getIFrameWrapper()?.classList.remove('fullscreen');
+  getIFrameWrapper()?.classList.add('content-viewer_iframe-wrapper');

```

Comment:
We should test this class being added and removed 🙂

Reply:
Would you like separate tests for these or should I add the check to the existing tests?

Reply:
Separate tests please. Each test should check just one thing 🙂